.github/workflows/lint-and-test.yml

```
@@ -0,0 +1,45 @@
+name: lint and test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  TZ: Asia/Tokyo
+
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v3
```

[nits]
最新がv5だったので
https://github.com/actions/setup-go
```suggestion
        uses: actions/setup-go@v5
```